core/src/documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,121 @@
+            embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring. 
+            In part thanks to the huge ecosystem of extensions that enhance and enrich the experience in VS Code, 
+            turning it into a super smart text editor, fast and lightweight. this guide focuses on the bare minimum that 
+            is needed to get started Spark source code with VS Code.
+        </p>
+
+        <h2>Prepare your environment</h2>
+        <p>
+            The rest of this guide will assume you have at least those parts installed:
+            <ul>
+                <li><b>JAVA Development Kit (version 8)</b> </li>
+                <li><b>Maven</b></li> 
+                <li><b>Copy of the  Spark source code available locally </b></li>
+                <li><b>Visual Studio Code</b></li>     
+            </ul>
+        <p>
+            Start by Adding Java language support to VS Code by installing the popular minimum requirement  Java extensions.
+            There is a Java Extension Pack, which contains the most popular extensions for most Java developers: 
+        </p>
+        <p>
+            Launch your VS Code and bring up extensions view by clicking on the <b>Extensions</b> icon in the Activity Bar
+            on the left side of VS Code or  <b>View > Extensions</b> then write the name of extension <b>Java Extension Pack</b> and 
+            then click on install link to install all extensions for Pack.
+        </p>
+        <figure>
+            <a href="images/VSCode-1.png"><img src="images/VSCode-1.png" alt="Installing Java Extension Pack to VS Code."></a>
+	    <figcaption>Installing Java Extension Pack to VS Code.</figcaption>            
+        </figure>
+
+        <h2>Open the source code folder</h2>
+        <p>
+            The Spark source code is using an Apache Maven project structure. As VS Code (with Maven Extension) recognizes this structure,
+            opening the source code into an VS Code project is very easy.
+        </p>
+        <p>
+            Select the 'Open' option -- <b>File > Open</b> and choose your spark source code folder then click on the open button. 
+            This will quickly import the Spark source code into  VS Code .
+        </p>
+
+        <figure>
+            <a href="images/VSCode-2.png"><img src="images/VSCode-2.png" alt="Use the 'Open' functionality of VS Code."></a>
+            <a href="images/VSCode-3.png"><img src="images/VSCode-3.png" alt="Select the folder that will open in our VS Code."></a>
+            <a href="images/VSCode-4.png"><img src="images/VSCode-4.png" alt="Overview of project in Visual Studio Code"></a>
+	    <figcaption>Opening Spark Source folder on VS Code.</figcaption>            
+        </figure>
+
+        <p>
+            VS Code will automatically recognize and process the Maven structure of Spark (pom.xml file) and it will download automatically all dependencies.
```

Comment:
I would change download automatically into automatically download. Sounds more English this way :)